contributor-dashboard-legacy/src/pages/SizeComparison.tsx

```diff
@@ -49,45 +23,23 @@ interface SizeSnapshot {
 	[bundleId: string]: { parsed: number; gzip: number };
 }
 
-async function fetchSizeSnapshotAzure(buildId: number): Promise<SizeSnapshot> {
-	const response = await fetch(
-		`https://dev.azure.com/mui-org/material-ui/_apis/build/builds/${buildId}/artifacts?api-version=5.1`
-	);
-
-	const body: AzureApiBody<AzureBuildArtifact[]> = await response.json();
-
-	if (response.status === 200) {
-		const artifacts = body.value;
-		const artifact = artifacts.find(
-			(artifact) => artifact.name === "size-snapshot"
-		);
-
-		const downloadUrl = new URL(artifact!.resource.downloadUrl);
-		downloadUrl.searchParams.set("format", "file");
-		downloadUrl.searchParams.set("subPath", "/size-snapshot.json");
-		return downloadSnapshot("size-snapshot-azure", downloadUrl.toString());
-	}
-
-	throw new Error(`${body.typeKey}: ${body.message}`);
-}
-
 async function fetchSizeSnapshotCircleCI(
-	buildNumber: number
+	buildNumber: number,
 ): Promise<SizeSnapshot> {
 	const response = await fetch(
-		`https://circleci.com/api/v2/project/gh/mui/material-ui/${buildNumber}/artifacts`
+		`/.netlify/functions/circle-ci-artifacts?buildNumber=${buildNumber}`,
```

Comment:
Just for good practice, avoid composing urls with what could be user input without proper encoding.

```suggestion
		`/.netlify/functions/circle-ci-artifacts?buildNumber=${encodeURIComponent(buildNumber)}`,
```


In my own code, I always prefer using `URL` for manipulating urls, I've spent way too much time in my life solving issues caused by false assumptions about encoding 🙂